pkg/operator/apiserver/controller/prune/prune_controller.go

```diff
@@ -0,0 +1,197 @@
+package prune
```

Comment:
is this controller specific to `apiserver` package?

Reply:
+1 for the question

Reply:
it's not, although it was only used for apiservers with deployments (no static pods) until now

Reply:
moved to pkg/operator among other similar controllers